f5_openstack_agent/lbaasv2/drivers/bigip/route.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+
+from f5_openstack_agent.lbaasv2.drivers.bigip.resource_helper \
+    import BigIPResourceHelper
+from f5_openstack_agent.lbaasv2.drivers.bigip.resource_helper \
+    import ResourceType
+
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+
+class RouteServiceBuilder(object):
+
+    def __init__(self):
+        self.route_manager = BigIPResourceHelper(ResourceType.route)
+
+    def create_route(self, bigip, payload):
+        if payload:
+            try:
+                self.route_manager.create(
+                    bigip, payload
+                )
+            except Exception as err:
+                if err.response.status_code == 409:
```

Comment:
If dirty data exists, why not overwrite it?

Ideally we should place those common code in a base class, instead of implement them every time.